src/components/layout/DialogHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from 'react';
+
+import { GraaspLogo } from '@graasp/ui';
+
+import { Stack, Typography, useTheme } from '@mui/material';
+
+type DialogHeaderProps = {
+  title: string;
+  description?: string | ReactNode;
+};
+export const DialogHeader = ({ title, description }: DialogHeaderProps) => {
+  const theme = useTheme();
+
+  return (
+    <Stack direction="column" alignItems="center" spacing={1} width="100%">
+      <GraaspLogo height={90} sx={{ fill: theme.palette.primary.main }} />
+      <Typography variant="h4" component="h2">
+        {title}
+      </Typography>
+      {description ? (
+        typeof description === 'string' ? (
+          <Typography pt={2}>{description}</Typography>
+        ) : (
+          description
+        )
+      ) : null}
```

Comment:
What about the following? If you used the ternary to avoid printing 0, you can wrap `Boolean(description)` to be sure.

```suggestion
      {description && (
        typeof description === 'string' ? (
          <Typography pt={2}>{description}</Typography>
        ) : (
          description
        )
      )}
```

Reply:
Good idea, I was not too fond of the nester ternary either but I did not find a better solution in the moment. I will apply yours !